packages/skipper/tests/FileSystem/PathNormalizerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\Skipper\Tests\FileSystem;
+
+use Iterator;
+use Symplify\PackageBuilder\Testing\AbstractKernelTestCase;
+use Symplify\Skipper\FileSystem\PathNormalizer;
+
+final class PathNormalizerTest extends AbstractKernelTestCase
+{
+
+    /**
+     * @dataProvider providePaths
+     */
+    public function testPaths(string $path, string $expected): void
+    {
+        $pathNormalizer = new PathNormalizer();
+
+        $this->assertEquals(
+            $expected,
+            $pathNormalizer->normalizeForFnmatch($path)
+        );
+    }
```

Comment:
I can't see `PathNormalizer` modifications, so I can't comment on them :/

Reply:
As mentioned in the description I don't have an idea how to modify jet.

Reply:
I see. Could you make this PR about adding only a passing test then? So we can merge it now and prevent stale. The test itself has value.

The fix should be a in new pull-request.

Reply:
I can do that.